docs/minor edits to principles (#267)
seedcase-project/seedcase-project: about/principles.qmd

    approach](https://community.seedcase-project.org/guides/documentation-first)
-   work and communicate openly (though privately where necessary)
-   use open licenses for all our outputs

> -   design documentation, training material, and software for both
>     people who are not computationally technical and people who are

Reviewer: ```suggestion
-   design documentation, training material, and software with
    non-technical people in mind 
```

> -   incorporate accessibility and inclusive language in all our outputs

Reviewer: ```suggestion
-   incorporate accessibility and inclusive design in both our software and documentation outputs
```
